models/BuddiesModel.js

```diff
@@ -0,0 +1,223 @@
+import { MongoClient } from 'mongodb';
+import dotenv from 'dotenv';
+import {
+  checkUserCache,
+  addUserToCache,
+  userCache,
+} from '../src/utils/userCache.js';
+import { SIMPLE_MODELS } from './SIMPLE_MODELS.js';
+
+dotenv.config();
+
+const client = new MongoClient(process.env.MONGO_URI);
+
+const { DB_NAME, COLLECTION_NAME } = SIMPLE_MODELS;
+
+// class User {
+// 	constructor(id) {
+// 		this._id = id,
+// 			this.timezone = null
+// 	}
+// }
+
+class PointsUser {
+  constructor() {
+    (this.pointsReceived = []),
+      (this.pointsGiven = []),
+      (this.lastPointsGivenBy = []);
+  }
+}
+
+class PointsObject {
+  constructor(giver, date, channel) {
+    (this.givenBy = giver), (this.date = date), (this.channel = channel);
+  }
+}
+
+class PointGivenBy {
+  constructor(userId, date) {
+    (this.userId = userId), (this.date = date);
+  }
+}
+
+export default class BuddiesModel {
```

Comment:
why export the db and use a bunch of static methods instead of instantiating and exporting the instance?

Reply:
Were not really exporting the DB, we are exporting a class that let's us work against it. It's just a tool that we can use to interact with the DB through a collection of static methods.